rocky-raft/src/rocky/raft/dto/LogEntry.java

```diff
@@ -10,17 +10,28 @@
 
     private String value;
 
+    private long id;
```

Comment:
Let's use String for ID. We can use Java's UUID instead of Random. UUID is theoretically more suitable for this purpose.